docker-compose.override.yml

```diff
@@ -7,7 +7,9 @@ services:
     command: ["./manage.py", "runserver", "0.0.0.0:8000"]
     # Log printing via Rich is enhanced by a TTY
     tty: true
-    env_file: ./dev/.env.docker-compose
+    env_file:
+      - ./dev/.env.docker-compose
+      - ./dev/.env.secrets
```

Comment:
Since `./dev/.env.secrets` is in the gitignore, instances without this file will not build. Is there a way this addition can be conditional? Or should we have a placeholder file?

Reply:
Good point. For now I'll remove this until we find some robust way to handle this.